packages/medusa/src/api/routes/admin/batch/index.ts

```diff
@@ -0,0 +1,41 @@
+import { Router } from "express"
+import { BatchJob } from "../../../.."
+import { DeleteResponse, PaginatedResponse } from "../../../../types/common"
+import middlewares from "../../../middlewares"
+
+export default (app) => {
+  const route = Router()
+
+  app.use("/batch", route)
+
+  route.get(
+    "/",
+    middlewares.normalizeQuery(),
+    middlewares.wrap(require("./list-batch-jobs").default)
+  )
```

Comment:
comment: I am assuming, that we'll wait with the sub-router pattern till next Batch PR, since `list-batch-jobs.ts` wont need it

Reply:
yes indeed, it was an artefact because it was the main branch for all the others and helped me having less conflict 😂